File: app/src/main/java/com/tkdev/weatherapp/presenter/current/CurrentPresenter.java
package com.tkdev.weatherapp.presenter.current;

import com.tkdev.weatherapp.presenter.MainContract;

public interface CurrentPresenter extends MainContract {

    String setTemperatureCurrentTextView();
    String setTemperatureMinimumTextView();
    String setTemperatureMaximumTextView();
    String setWeatherDescriptionTextView();
    String setHumidityViewText();
    String setLastUpdateViewText();
    String setDateViewText();
    String setCityName();

    String sendCurrentWeather();
}

File: app/src/main/java/com/tkdev/weatherapp/presenter/current/CurrentPresenter.kt
package com.tkdev.weatherapp.presenter.current

import com.tkdev.weatherapp.presenter.MainContract

interface CurrentPresenter : MainContract {
    fun setTemperatureCurrentTextView(): String
    fun setTemperatureMinimumTextView(): String
    fun setTemperatureMaximumTextView(): String
    fun setWeatherDescriptionTextView(): String
    fun setHumidityViewText(): String
    fun setLastUpdateViewText(): String
    fun setDateViewText(): String
    fun setCityName(): String
    fun sendCurrentWeather(): String
}